build_config/lib/src/build_config.dart

```diff
@@ -109,7 +109,7 @@ class BuildConfig {
         (map) =>
             BuildConfig.fromMap(packageName, dependencies, map ?? const {}),
         allowNull: true,
-        sourceUrl: configYamlPath,
+        sourceUrl: Uri.file(configYamlPath),
```

Comment:
This can be null it looks like, at least in tests

Reply:
yup...

Reply:
fixed